include/scheduler.hpp

```diff
@@ -0,0 +1,125 @@
+#ifndef GRAPH_PROTOTYPE_SCHEDULER_HPP
+#define GRAPH_PROTOTYPE_SCHEDULER_HPP
+#include <graph.hpp>
+#include <set>
+#include <queue>
+
+namespace fair::graph::scheduler {
```

Comment:
Like it makes it more readable further below:
`gr::scheduler::simple(...)`
vs.
`gr::scheduler::breadth_first(...)`

:+1: